src/mapcraftercore/renderer/renderviews/isometric/tilerenderer.cpp

```diff
@@ -194,17 +194,16 @@ void IsometricTileRenderer::renderTile(const TilePos& tile_pos, RGBAImage& tile)
 			// get local block position
 			mc::LocalBlockPos local(block.current);
 
-			// now get block id
-			uint16_t id = current_chunk->getBlockID(local);
+			// now get block id and block data
+			uint16_t id, data, extra_data;
+			current_chunk->getBlockInfo(local, id, data, extra_data);
```

Comment:
For the small test I used, not really, but I expect that it might have more influence on larger worlds (given that most worlds container more air than any other "block").

It can be avoided if the id is retrieved, as before... but as `getBlockExtraData` needs the id, it needs to be passed as an argument to it, which is not really good code quality in my opinion. Hiding it behind `Chunk::getBlockInfo` seemed a decent solution.

But as performance for these kind of programs is important, I'll revert back to the previous implementation, and will pass the id to `Chunk::getBlockExtraData`. Or `Chunk::getBlockExtraData` first needs to retrieve the id, which is probably going to be even slower (twice the same access for the id), so let's not do that.